packages/manager/apps/pci-instances/src/adapters/tanstack/instances/queryKeys.ts

```diff
@@ -31,3 +31,9 @@ export const instanceQueryKeyPredicate = (
     'instance',
     instanceId,
   ]).every((elt) => query.queryKey.includes(elt));
+
+export const instanceCreationMutationKey = (projectId: string): string[] => [
```

Comment:
Maybe we could move this to a `mutationKeys` file to prevent from mixing concepts